python/mxnet/pretrained_model.py

```diff
@@ -79,16 +79,16 @@ def download_model(url, location=DEFAULT_MODEL_LOCATION, overwrite=False):
     Parameters
     ----------
     url : str,
-      URL of the model. Get the list of available models from https://github.com/dato-code/mxnet.
+      URL of the model. Get the list of available models from https://s3.amazonaws.com/dato-models/mxnet_models/release/
```

Comment:
Lets link to GLC docstring here. https://dato.com/products/create/docs/graphlab.mxnet.pretrained_image_model.html